individual-reflection.md

```diff
@@ -8,65 +8,86 @@ Answer the following comprehension questions **within this file.** Write your an
 
 ### `GET` Request Review
 
-1. Describe a GET request that your project makes, and the high-level description of what it does
+1.  Describe a GET request that your project makes, and the high-level description of what it does
+    - Answer: A Get request for my project is requesting that my browser perform an action for example. I created a method that requests a url and sub url to the user list or channel list.
+1.  What is the verb of this request?
+    - Answer: GET
+1.  What is the path (or the URL, or endpoint) of this request?
+    - Answer: "https://slack.com/api/#{sub_url}?token=#{SLACK_TOKEN}&pretty=1"
+1.  What are the query params (the additional data sent with the request, besides the verb and the path)?
     - Answer:
-1. What is the verb of this request?
-    - Answer:
-1. What is the path (or the URL, or endpoint) of this request?
-    - Answer:
-1. What are the query params (the additional data sent with the request, besides the verb and the path)?
-    - Answer: 
-1. What is the syntax used to make this request? (Copy and paste a code snippet here)
+1.  What is the syntax used to make this request? (Copy and paste a code snippet here)
+
     - Answer:
-      ```ruby
-      # Copy and paste your answer below this comment
+      ```ruby # Copy and paste your answer below this comment
+      def self.get(sub_url) # self calls only know about its self to call in another method # must call the (Class name).something # Recipent.get(querty) is called on the respective class that
+      url = "https://slack.com/api/#{sub_url}?token=#{SLACK_TOKEN}&pretty=1"
+      response = HTTParty.get(url)
```

Comment:
This is a perfectly good way to make this HTTP call! You construct the `url` by interpolating a lot of the important things you need, such as the `sub_url` and API key.

In case the string interpolation is confusing to read, it's good to know that the API key is being passed in as a query parameter. (One way to tell this by reading it is seeing any URL that has a `?` in it, and seeing the different pairs afterwards. In this example, we see `token=...`)

If you wanted to try an alternative syntax, you may use this syntax with HTTParty:

```ruby
HTTParty.get("https://slack.com/api/#{sub_url}", query: {token: ENV['SLACK_TOKEN']})
```

This line is saying "Make a GET request to this url. Also, pass along these query parameters: `token`, with a value of `ENV[SLACK_TOKEN]`"

Again, both syntaxes are correct; just wanted to make sure you knew the other option!